views/page-not-found.html

```diff
@@ -0,0 +1,20 @@
+{% extends "layout.html" %}
+
+{% block pageTitle %}
+  Page not found - File a confirmation statement
+{% endblock %}
+{% block signOut %}{% endblock %}
```

Comment:
a bit late I know, but line 6 can be removed - the signOut block we aren't using in out layout.html so we don't need to override it with an empty block. We may add it in future but don't seem to be using it yet.